app/controllers/admin/base_controller.rb

```diff
@@ -1,7 +1,10 @@
 module Admin
   # Admin::BaseController
   class BaseController < ::ApplicationController
-    before_action :authenticate_admin_user!
+    include Accessible
+
+    skip_before_action :check_user, only: :destroy
```

Comment:
__[RuboCop]__

Rails/LexicallyScopedActionFilter: `destroy` is not explicitly defined on the controller.


<!-- Comment by [SideCI](https://sideci.com) -->